part2.kt

```diff
@@ -0,0 +1,9 @@
+fun main(args: Array<String>) {
+
+    var string1="The quick brown fox jumps over the lazy dog"
+    var string2= "brown fox jumps"
+
+    println(string2.split(string1))

```

Comment:
try to use substring here

Reply:
fun main(){

    var old =" The quick brown fox jumps over the lazy dog."

    var new : String

    println(old.substring(11,26))
}